benchmarks/integer_overflow_and_underflow/overflow_add.yaml

```diff
@@ -0,0 +1,8 @@
+integer_underflow_and_overflow/overflow_add:
+  has_bug: true  # Trail of bits suggests adding a test where has_bug: false
+  issues:
+    - location:
+        - bytecode_offset: 139
+          line_number: 15
+          src: 9:27  # or 9:27:0 to match solc ast?
+          vulnerability: Overflow
```

Comment:
Why do we need this?

Reply:
If you look at the current benchmark resutls say https://ethereumanalysisbenchmarks.github.io/nssc/index.html#honeypots/GiftBox/GiftBox you will see that there are multiple vulnerabilities for a single location.

Reply:
Funny example. Oyente reports both an integer overflow and integer underflow for the same location, both of which are false positives.

Reply:
Adding this should be fine as long as it's not mandatory. Maybe it's useful in some cases.